airbyte-to-flow/src/connector_runner.rs

```diff
@@ -150,14 +125,14 @@ pub async fn run_airbyte_source_connector(
         }
 
         exit_status_result
-    });
+    };
```

Comment:
There were some changes made in https://github.com/estuary/airbyte/pull/54 to the code block around `if exit_status_result.is_ok() {` (above) specifically for connectors using TCP. Do we not need to undo those changes as well with the switch back to stdio?

Reply:
I think it makes sense to keep the new code as-is - just wasn't sure if it was equivalent or not. I agree that it seems like a nice improvement.